### DISCUSSION TITLE
MEETING: ecatalogue Working Group - 3 May 2023

### DISCUSSION THREAD

**Discussion by fmjjones** [fieldmuseum/EMu-Documentation#822] [MEETINGS]
# Agenda: Catalogue Working Group Part Deux
[Github Discussion from Part 1](https://github.com/fieldmuseum/EMu-Documentation/discussions/821)

- [Measurement Taken](https://docs.google.com/spreadsheets/d/19_qLQL-aTkWN-Y6cv0S2IevpoQ6bSnDxBxhHpNlxazE/edit#gid=496435033)

Field Help Definition: Dimensions or counts to define the “kind” of measurement (e.g. Total Length, Width, Height,)
Ant: e.g. “Overall”, “Approximate”, “Image”, sometimes “Confirmed” for exhibits
Mam: e.g. “Tail length” (could pull from anatomical vocab)
Fish: Meas. table includes Prep Type. used similarly to Mammals (e.g. “Standard Length”, etc–description of what’s measured)

And define data being recorded
…

From current field-help:
How the measurement was taken. Ex: Estimate, Approximate, Overall, Confirmed etc.
For values to display in the corresponding exhibitions object module record this field should be set to "Confirmed". Note that ONLY where Measurement Kind = length, height, depth and weight are currently mapped.

Kind of measurement. Ex: depth, diameter, height, weight, age etc. Choose from previously used values in lookup list, or enter a new value.

For values to display in the corresponding exhibitions object module record the Measurement taken field must also be set to "Confirmed". Note that ONLY where Measurement Kind = length, height, depth and weight are currently mapped.

From Exhib.Objs (eexhibitobjects.MeaMeasurementTaken_tab)
This field describes the aspect of the object that was measured.
Examples: Framed, or Unframed, Base, Object Only, or With Base, Open (for a Book)
NOTE: Information that qualifies the measurement, like Estimated, Confirmed, Calculated, or From Memory belongs in "Remarks."

- [Measurement Kind](https://docs.google.com/spreadsheets/d/19_qLQL-aTkWN-Y6cv0S2IevpoQ6bSnDxBxhHpNlxazE/edit#gid=178476466)
Field Help Definition:
New Definition: Use this field to indicate specimen/parts measurement was taken from. Anth will be using “Overall”and “Part”

Ant: e.g. “Height”, “Length”, etc (which measurement; not currently a lookup list)

Mam: e.g. “Linear”, “Calipers”
Need data cleanup (some values wandered into Meas.Kind)
Fish: does not have this field currently. Will use this field to indicate Prep Type. (Prep Type is currently loaded in prep type lookup)

Exh Obj Definition - MeaCatalogueDepth
NOTE: There is a bug in this field whereby it is displaying depth instead of width - This has been reported to KE Feb2016 for a fix.
This value is derived from the attached catalogue record.
For data to display here these values must be filled in in the catalogue record.
"Measurement taken" = Confirmed
"Measurement kind" = width

And define data being recorded
…

Example walk-thru:
Botany doesn't have a ton of data in these so we are feeling pretty flexible, as long as we have clearly defined landing places at the end for our pieces of data.  ie 4 fields for [a blue] "Spore" / "length" / "6.2" / "μm" [measured by WG]
Meas. Taken = “Length”
Meas. Kind = “Spore” 
Unit = μm
Value = 0.6
Meas. By = Wyatt Gaswick
Feature = Blue
Confirmed = (“Yes” if someone confirmed the accuracy of this measurement – primarily used by Anthro & Exhibits)

Other Measurements notes:
Units – if new units need to be added, they can be.
Measurement values do not need to be metric
Mapping Type & Kind to MeasOrFact extension - https://rs.gbif.org/extension/obis/extended_measurement_or_fact.xml 

- Preps and Parts (and indicators’ logic)

Field Help Definition:
Only check Alc, not ASR, but for ASR check ASR and Skull.
LotWet indicator: Keep current logic, but add PrepType=”Wet” to the logic.
Ins - has some specimens in non-alcohol solution; uses ‘Preservation’ to indicate exact solution + ‘wet’ (in Prep Type)
IZ - most (but not all)l wet collections = in alcohol
DECISION: Add if Prep = ‘wet’, check ‘wet’ indicator

Does everyone need specifics on hand – wet vs dry (vs specific solutions)
For indicators
For other Prep/Part lookup values

# Homework
Other Indicator logic
To be cont’d

Other # Source (Catalog Tab)
Field Help Definition

[From Group Testing](https://docs.google.com/spreadsheets/d/1MUg92tIPsuuhkWAQOM4iIB4nwJV_z2oM4oCq4b7PB04/edit?pli=1#gid=1658218035)

**Comment by rondlg** [comment]
**Measurements** came up in a separate conversation about Minerals and so I've started to gather details of what we do now from the field level help in an FAQ. As this topic starts to align between ecatalogue and eexhibit objects we can refine and update as necessary.

[How do we handle measurements, in concept anyway! FAQ](https://github.com/fieldmuseum/EMu-Documentation/discussions/837)
